### PR TITLE
corei7 arch by default for containers

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
- also minor sandbox updates for rebuilding dpdk and python3'isms

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
waiting to see on the ffi/capsule-side what may be needed here when @drunkirishcoder compiles things via haswell-instruction set.

**Comment by zeeshanlakhani** [comment]
further reading, dpdk configs & vpp, showcase corei7 is the best default to work across arches. @drunkirishcoder double-check with this branch to see if it works on your side.

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,10 @@ FROM $BUILDER_BASE_IMG as builder
 
 ARG DPDK_VERSION
 ARG DEBUG=false
+ARG DPDK_MACHINE=corei7
```
Comment (drunkirishcoder) [inline Dockerfile:8]: can you move these 2 args next to each other?

File: docker.mk [hunk 2]
```diff
@@ -4,9 +4,11 @@ DOCKER_NAMESPACE = getcapsule
 
 DPDK_IMG = dpdk
 DPDK_DEVBIND_IMG = dpdk-devbind
+DPDK_MACHINE = corei7
```
Comment (drunkirishcoder) [inline docker.mk:7]: do we need to specify here? they are defaulted in the dockerfile and we are not changing them in makefile.

**Pull request merged**